website/static/js/nodeControl.js

```diff
@@ -35,7 +35,15 @@ var MESSAGES = {
 
     makeComponentPrivateWarning: 'Making a component private will prevent users from viewing it on this site, ' +
                         'but will have no impact on external sites, including Google\'s cache. ' +
-                        'Would you like to continue?'
+                        'Would you like to continue?',
+
+    // TODO(hrybacki): Remove once the hotifx is taken out
+    makePostCutoffRegistrationPublicWarning: 'Once a project is made public, there is no way to guarantee that ' +
```

Comment:
What lines are you asking to be reviewed? makePostCutoffRegistrationPublicWarning?